src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Collections/SegmentedArrayHelper.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.Shared.Collections
+{
+    internal static class SegmentedArrayHelper
+    {
+        internal static int CalculateSegmentSize(int elementSize)
```

Comment:
What is elementSize for? It's not used

Reply:
It's a multiplier for segmentSize, but I forgot to use it.

Reply:
Fixed